src/view-backend.c

```diff
@@ -65,6 +65,9 @@ wpe_view_backend_destroy(struct wpe_view_backend* backend)
     backend->input_client = 0;
     backend->input_client_data = 0;
 
+    backend->fullscreen_client = 0;
+    backend->fullscreen_client_data = 0;
```

Comment:
I'm somewhat amused that this does not cause compiler warnings; we should be using `NULL` here because the members are pointers 🤔. Anyhoo, let's use it:

```suggestion
    backend->fullscreen_client = NULL;
    backend->fullscreen_client_data = NULL;
```

Reply:
Done. I normally use NULL (or nullptr in C++11) but followed the style of the other setters which were using 0.